app/assets/javascripts/backbone/views/loan_questions_view.coffee

```diff
@@ -130,7 +131,12 @@ class MS.Views.LoanQuestionsView extends Backbone.View
   showHideAssociations: (e) ->
     overrideParent = e.currentTarget
 
+    # TODO: Using show/hide preferred, if possible
```

Comment:
Using show/hide may depends on what method is chosen for hiding/showing the table on load based on override_associations value.